tool/templates/dart/struct.dart.jinja

```diff
@@ -18,6 +18,11 @@ final class {{type_name}} {
   {% if !mutable -%} final {% endif -%} {{field.dart_type_name}} {{field.name}};
   {%- endfor %}
 
+  {%- for lifetime in lifetimes %}
+  // ignore: unused element
+  final core.List<Object> _edge{{lifetime}};
```

Comment:
issue: this isn't correct: these lifetimes can change as the struct is mutated. We compute them on the fly with _fields_for_lifetime for this reason

Reply:
that approach doesn't work for slices, where we need to hold on to the allocated copy. We'll need another approach